static_camera_config: # configs around world frame
  camera_1:
    nickname: 'alpha'
    resolution: [640, 360] # must be integer
    pose:
      position:
        x: 0.0  # Position in meters (x-axis) [float]
        y: -3.0   # Position in meters (y-axis) [float]
        z: 1.0  # Position in meters (z-axis) [float]
      orientation:
        roll: -1.57 # alpha in radians
        pitch: 0.0  # beta in radians
        yaw: -3.14  # gamma in radians
  camera_2:
    nickname: 'beta'
    resolution: [640, 360] # must be integer
    pose:
      position:
        x: 3.0  # Position in meters (x-axis) [float]
        y: 0.0   # Position in meters (y-axis) [float]
        z: 1.0  # Position in meters (z-axis) [float]
      orientation:
        roll: -1.57 # alpha in radians
        pitch: -1.57  # beta in radians
        yaw: -3.14  # gamma in radians
  camera_3:
    nickname: 'gamma'
    resolution: [640, 360] # must be integer
    pose:
      position:
        x: -3.0  # Position in meters (x-axis) [float]
        y: 3.0   # Position in meters (y-axis) [float]
        z: 1.0  # Position in meters (z-axis) [float]
      orientation:
        roll: 1.57 # alpha in radians
        pitch: 0.785 # beta in radians
        yaw: 0.0  # gamma in radians
# https://dugas.ch/transform_viewer/index.html
scene_config:
  scene_file: 'assets/scenes/default.ttt'
  sim_timestep: 0.010
